FoodPin/FoodPin/Controller/RestaurantDetailViewController.swift

```diff
@@ -33,15 +37,41 @@ class RestaurantDetailViewController: UIViewController {
     func setRestaurat(restaurant: Restaurant) {
         self.restaurant = restaurant
     }
+}
 
-    /*
-    // MARK: - Navigation
+extension RestaurantDetailViewController: UITableViewDataSource, UITableViewDelegate {
 
-    // In a storyboard-based application, you will often want to do a little preparation before navigation
-    override func prepare(for segue: UIStoryboardSegue, sender: Any?) {
-        // Get the new view controller using segue.destination.
-        // Pass the selected object to the new view controller.
+    func tableView(_ tableView: UITableView, numberOfRowsInSection section: Int) -> Int {
+        return 2
     }
-    */
 
+    func tableView(_ tableView: UITableView, cellForRowAt indexPath: IndexPath) -> UITableViewCell {
+        switch indexPath.row {
+        case 0:
+            guard let cell = tableView.dequeueReusableCell(withIdentifier:
+                                                            String(describing: RestaurantDetailTextCell.self),
+                                                           for: indexPath) as? RestaurantDetailTextCell else {
+                return UITableViewCell()
+            }
+
+            cell.setDescriptionLabel(text: restaurant.getDescrption())
+            return cell
+
+        case 1:
+            guard let cell = tableView.dequeueReusableCell(withIdentifier:
+                                                            String(describing: RestaurantDetailTwoColumnCell.self),
+                                                           for: indexPath) as? RestaurantDetailTwoColumnCell else {
+                return UITableViewCell()
+            }
+
+            cell.setColumn1TitleLabel(text: "Address")
+            cell.setColumn1TextLabel(text: restaurant.getLocation())
+            cell.setColumn2TitleLabel(text: "Phone")
+            cell.setColumn2TextLabel(text: restaurant.getPhone())
```

Comment:
I think you can also have all this under one method where you send the restaurant object